client/state/automated-transfer/constants.js

```diff
@@ -1,7 +1,16 @@
 export const AUTOMATED_TRANSFER_STATUS = {
+	INQUIRING: 'inquiring',
 	START: 'start',
 	SETUP: 'setup',
 	LEAVING: 'leaving',
 	CONFLICTS: 'conflicts',
 	COMPLETE: 'complete',
 };
+
+export const eligibilityHolds = {
+	NO_WPCOM_NAMESERVERS: 'NO_WPCOM_NAMESERVERS',
+	NON_ADMIN_USER: 'NON_ADMIN_USER',
+	NOT_USING_CUSTOM_DOMAIN: 'NOT_USING_CUSTOM_DOMAIN',
+	SITE_GRAYLISTED: 'SITE_GRAYLISTED',
+	SITE_PRIVATE: 'SITE_PRIVATE',
+};
```

Comment:
I don't have a strong opinion about it, and as we don't have a rule, I'd say let's go your way.